demisto_sdk/commands/create_artifacts/content_artifacts_creator.py

```diff
@@ -329,7 +511,8 @@ def dump_packs(artifact_manager: ArtifactsManager, pool: ProcessPool) -> List[Pr
     """
     futures = []
     for pack_name, pack in artifact_manager.content.packs.items():
-        if pack_name not in IGNORED_PACKS:
+        if (pack_name in artifact_manager.pack_names or 'all' in artifact_manager.pack_names) and \
```

Comment:
instead of iterating the whole content and search for the desired pack, maybe switch to iterate over the given pack names. if `all` is specified - keep the current logic.

check how fast it runs for 2 packs, if not significant - you can leave it as is.